### PR TITLE
[Issue #95707] Remove WindowsGMSA and WindowsRunAsUserName features f…

### PR DESCRIPTION
**Pull request by mrmassis**
WindowsGMSA and WindowsRunAsUserName features, and they traces removed from code.
/kind cleanup

### CODE REVIEW

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -297,7 +296,7 @@ const (
 	CSIStorageCapacity featuregate.Feature = "CSIStorageCapacity"
 
 	// owner: @alculquicondor
-	// beta: v1.20
+	// alpha: v1.19
```
Comment (marosset) [inline pkg/features/kube_features.go:299]: Can you revert your changes this line?

File: pkg/features/kube_features.go [hunk 2]
```diff
@@ -678,7 +662,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	StorageObjectInUseProtection:   {Default: true, PreRelease: featuregate.GA},
 	SupportPodPidsLimit:            {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.21
 	SupportNodePidsLimit:           {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.21
-	HyperVContainer:                {Default: false, PreRelease: featuregate.Deprecated},
+	HyperVContainer:                {Default: false, PreRelease: featuregate.Alpha},
```
Comment (marosset) [inline pkg/features/kube_features.go:665]: This change also needs to be reverted in your PR

Comment (mrmassis) [reply pkg/features/kube_features.go:665]: Ok

File: pkg/features/kube_features.go (hunk 1 above)
Comment (mrmassis) [reply pkg/features/kube_features.go:299]: Ok, sorry!

File: pkg/features/kube_features.go (hunk 2 above)
Comment (mrmassis) [reply pkg/features/kube_features.go]: Done, i revert this entries and did a push. I must done other PR?

Comment (mrmassis) [reply pkg/features/kube_features.go]: I used with base the branch master. Is correct?

**Pull request closed without merging**